### ISSUE TITLE
Quarkus 3.10 and 1.3.2

### ISSUE THREAD

**Issue by oleg-nenashev** [quarkiverse/quarkus-wiremock#128]
On Java 17, the following error happens. It is similar to https://github.com/quarkusio/quarkus/issues/30346 and can be reproduced in https://github.com/oleg-nenashev/gradle-quarkus-plugin-demo/pull/36 

```
* What went wrong:
Execution failed for task ':quarkusAppPartsBuild'.
> There was a failure while executing work items
   > A failure occurred while executing io.quarkus.gradle.tasks.worker.BuildWorker
      > io.quarkus.builder.BuildException: Build failure: Build failed due to errors
                [error]: Build step io.quarkiverse.wiremock.devservice.WireMockServerProcessor#setup threw an exception: javax.xml.transform.TransformerFactoryConfigurationError: Provider org.apache.xalan.processor.TransformerFactoryImpl not found
                at javax.xml.transform.TransformerFactory.newInstance(Unknown Source)
                at com.github.tomakehurst.wiremock.common.xml.Xml.optimizeFactoriesLoading(Xml.java:50)
                at com.github.tomakehurst.wiremock.core.WireMockApp.<init>(WireMockApp.java:77)
                at com.github.tomakehurst.wiremock.WireMockServer.<init>(WireMockServer.java:74)
                at io.quarkiverse.wiremock.devservice.WireMockServerProcessor.startWireMockDevService(WireMockServerProcessor.java:116)
                at io.quarkiverse.wiremock.devservice.WireMockServerProcessor.setup(WireMockServerProcessor.java:77)
                at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
                at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                at java.base/java.lang.reflect.Method.invoke(Method.java:568)
                at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:858)
                at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
                at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
                at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
                at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
                at java.base/java.lang.Thread.run(Thread.java:840)
                at org.jboss.threads.JBossThread.run(JBossThread.java:501)

```

**Comment by chberger** [comment]
Could you please test it again with `1.3.3`

**Comment by chberger** [comment]
@oleg-nenashev `Quarkus 3.10` wasn't officially supported by version `1.3.2`. I’ve just released version `1.4.0`, which officially supports `Quarkus 3.15 LTS`. Could you please try again and see if this works four you?
